addons/mail/static/src/components/attachment_box/attachment_box.scss

```diff
@@ -1,46 +1,7 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
-.o_AttachmentBox {
-    position: relative;
-}
-
-.o_AttachmentBox_buttonAdd {
-    align-self: center;
```

Comment:
This rule was not added in template, is it a default value? And did it not come in override of another value?

Reply:
@seb-odoo , this line was not necessary because the button fills the entire available width.